### ISSUE TITLE
Suddenly Dropping Frames Like Crazy

### ISSUE THREAD

**Issue by TideGear** [libretro-mirrors/beetle-saturn-libretro#206]
I've been able to run the Beetle Saturn core on my Windows PC full-speed 60 FPS without any problems for a while now, but it seems like the latest build is dropping frames like mad even during the Saturn boot-up animation. I didn't change anything except for updating my cores. I'm only trying to play the first Panzer Dragoon!

**Comment by TideGear** [comment]
Ok, I narrowed down that it seems to be related to using Hard GPU Sync. I turn that off, and suddenly it's fine. Leaving on Hard GPU Sync and turning on Adaptive VSync seems to almost fix it too, but it's not ideal.

I don't understand, since I was able to use Hard GPU Sync without issue with previous builds. I tried another game that ran perfectly before (Gungriffon), and it also slows down now too.

Seems the most recent build (or builds) broke something.

![retroarch 2021-07-19 15-53-31-08](https://user-images.githubusercontent.com/16357059/126238191-7253f0fc-ca6e-43a1-95c5-125052a12ddb.png)

**Comment by TideGear** [comment]
I fixed it by changing this from the default of "Optimal power" to "Prefer maximum performance". "Adaptive" didn't seem to work. I'm back to 60 FPS!
![image](https://user-images.githubusercontent.com/16357059/126267107-d4044dfa-75df-4f0a-97a9-6daab9dac9f9.png)

I don't know if this means there was no issue to begin with or not, but I'm not sure what changed to make the frames drop. Anyway, I fixed it for myself, so I'll close this for now.
